GeneralsMD/Code/GameEngine/Source/GameNetwork/GameSpy/MainMenuUtils.cpp

```diff
@@ -316,7 +316,7 @@ static void queuePatch(Bool mandatory, AsciiString downloadURL)
 ///////////////////////////////////////////////////////////////////////////////////////
 
 static GHTTPBool motdCallback( GHTTPRequest request, GHTTPResult result,
-															char * buffer, int bufferLen, void * param )
+															char * buffer, GHTTPByteCount bufferLen, void * param )
```

Comment:
This should be part of another PR

Reply:
If we merge with rebase then I think it is fine. The commits will be separate.